# adventr

<details>

* Version: 0.1.8
* GitHub: NA
* Source code: https://github.com/cran/adventr
* Date/Publication: 2020-05-05 16:50:06 UTC
* Number of recursive dependencies: 165

Run `revdepcheck::revdep_details(, "adventr")` for more info

</details>

## In both

*   checking whether package ‘adventr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘learnr’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/adventr/new/adventr.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        tutorials   5.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BayesFactor’ ‘GGally’ ‘Hmisc’ ‘WRS2’ ‘boot’ ‘car’ ‘dplyr’ ‘effects’
      ‘effsize’ ‘forcats’ ‘ggplot2’ ‘lm.beta’ ‘nlme’ ‘readr’ ‘robust’
      ‘sandwich’ ‘sjstats’ ‘tidyr’
      All declared Imports should be used.
    ```

# airGR

<details>

* Version: 1.7.6
* GitHub: NA
* Source code: https://github.com/cran/airGR
* Date/Publication: 2023-10-26 07:30:05 UTC
* Number of recursive dependencies: 110

Run `revdepcheck::revdep_details(, "airGR")` for more info

</details>

## In both

*   checking whether package ‘airGR’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/ggally/revdep/checks.noindex/airGR/new/airGR.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘airGR’ ...
** package ‘airGR’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
using C compiler: ‘Apple clang version 15.0.0 (clang-1500.1.0.2.5)’
sh: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
using SDK: ‘MacOSX14.2.sdk’
clang -arch arm64 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/opt/R/arm64/include -I/usr/local/include -Xclang -fopenmp    -fPIC  -falign-functions=64 -Wall -g -O2  -c airGR.c -o airGR.o
/opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran  -fPIC  -Wall -g -O2  -c  frun_CEMANEIGE.f90 -o frun_CEMANEIGE.o
make: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
...
make: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
make: *** [frun_CEMANEIGE.o] Error 1
make: *** Waiting for unfinished jobs....
make: *** [frun_GR4H.o] Error 1
make: *** [frun_GR2M.o] Error 1
make: *** [frun_GR1A.o] Error 1
make: *** [frun_GR4J.o] Error 1
make: *** [frun_GR5H.o] Error 1
ERROR: compilation failed for package ‘airGR’
* removing ‘/ggally/revdep/checks.noindex/airGR/new/airGR.Rcheck/airGR’


```
### CRAN

```
* installing *source* package ‘airGR’ ...
** package ‘airGR’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
using C compiler: ‘Apple clang version 15.0.0 (clang-1500.1.0.2.5)’
sh: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
using SDK: ‘MacOSX14.2.sdk’
clang -arch arm64 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/opt/R/arm64/include -I/usr/local/include -Xclang -fopenmp    -fPIC  -falign-functions=64 -Wall -g -O2  -c airGR.c -o airGR.o
/opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran  -fPIC  -Wall -g -O2  -c  frun_CEMANEIGE.f90 -o frun_CEMANEIGE.o
make: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
...
make: *** Waiting for unfinished jobs....
make: *** [frun_GR1A.o] Error 1
make: *** [frun_GR2M.o] Error 1
make: *** [frun_GR4H.o] Error 1
make: *** [frun_GR5H.o] Error 1
make: *** [frun_GR4J.o] Error 1
make: *** [frun_GR6J.o] Error 1
make: *** [frun_GR5J.o] Error 1
ERROR: compilation failed for package ‘airGR’
* removing ‘/ggally/revdep/checks.noindex/airGR/old/airGR.Rcheck/airGR’


```
# alevinQC

<details>

* Version: 1.18.0
* GitHub: https://github.com/csoneson/alevinQC
* Source code: https://github.com/cran/alevinQC
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 110

Run `revdepcheck::revdep_details(, "alevinQC")` for more info

</details>

## In both

*   checking C++ specification ... NOTE
    ```
      Specified C++11: please drop specification unless essential
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 20.5Mb
      sub-directories of 1Mb or more:
        extdata  19.2Mb
    ```

# AlpsNMR

<details>

* Version: 4.4.0
* GitHub: https://github.com/sipss/AlpsNMR
* Source code: https://github.com/cran/AlpsNMR
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 187

Run `revdepcheck::revdep_details(, "AlpsNMR")` for more info

</details>

## In both

*   checking whether package ‘AlpsNMR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘future’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/AlpsNMR/new/AlpsNMR.Rcheck/00install.out’ for details.
    ```

# AntMAN

<details>

* Version: 1.1.0
* GitHub: https://github.com/bbodin/AntMAN
* Source code: https://github.com/cran/AntMAN
* Date/Publication: 2021-07-23 10:00:02 UTC
* Number of recursive dependencies: 80

Run `revdepcheck::revdep_details(, "AntMAN")` for more info

</details>

## In both

*   checking whether package ‘AntMAN’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘mvtnorm’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/AntMAN/new/AntMAN.Rcheck/00install.out’ for details.
    ```

*   checking C++ specification ... NOTE
    ```
      Specified C++11: please drop specification unless essential
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rdpack’ ‘mcclust’
      All declared Imports should be used.
    ```

# BasketballAnalyzeR

<details>

* Version: 0.5.0
* GitHub: https://github.com/sndmrc/BasketballAnalyzeR
* Source code: https://github.com/cran/BasketballAnalyzeR
* Date/Publication: 2020-06-26 09:00:11 UTC
* Number of recursive dependencies: 86

Run `revdepcheck::revdep_details(, "BasketballAnalyzeR")` for more info

</details>

## In both

*   checking whether package ‘BasketballAnalyzeR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/BasketballAnalyzeR/new/BasketballAnalyzeR.Rcheck/00install.out’ for details.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘circlize’ ‘hexbin’ ‘scales’ ‘sna’
      All declared Imports should be used.
    ```

# BindingSiteFinder

<details>

* Version: 2.0.0
* GitHub: https://github.com/ZarnackGroup/BindingSiteFinder
* Source code: https://github.com/cran/BindingSiteFinder
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 218

Run `revdepcheck::revdep_details(, "BindingSiteFinder")` for more info

</details>

## In both

*   checking whether package ‘BindingSiteFinder’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 4.3.1
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
      Warning: package ‘S4Vectors’ was built under R version 4.3.2
      Warning: package ‘IRanges’ was built under R version 4.3.1
      Warning: package ‘GenomeInfoDb’ was built under R version 4.3.2
    See ‘/ggally/revdep/checks.noindex/BindingSiteFinder/new/BindingSiteFinder.Rcheck/00install.out’ for details.
    ```

# bootcluster

<details>

* Version: 0.3.2
* GitHub: NA
* Source code: https://github.com/cran/bootcluster
* Date/Publication: 2022-01-29 22:50:03 UTC
* Number of recursive dependencies: 82

Run `revdepcheck::revdep_details(, "bootcluster")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘sna’
      All declared Imports should be used.
    ```

# CARBayes

<details>

* Version: 6.1
* GitHub: https://github.com/duncanplee/CARBayes
* Source code: https://github.com/cran/CARBayes
* Date/Publication: 2023-11-17 11:40:11 UTC
* Number of recursive dependencies: 139

Run `revdepcheck::revdep_details(, "CARBayes")` for more info

</details>

## In both

*   checking whether package ‘CARBayes’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘MASS’ was built under R version 4.3.1
      Warning: package ‘Rcpp’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/CARBayes/new/CARBayes.Rcheck/00install.out’ for details.
    ```

# CARBayesST

<details>

* Version: 4.0
* GitHub: https://github.com/duncanplee/CARBayesST
* Source code: https://github.com/cran/CARBayesST
* Date/Publication: 2023-10-30 16:40:02 UTC
* Number of recursive dependencies: 119

Run `revdepcheck::revdep_details(, "CARBayesST")` for more info

</details>

## In both

*   checking whether package ‘CARBayesST’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘MASS’ was built under R version 4.3.1
      Warning: package ‘Rcpp’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/CARBayesST/new/CARBayesST.Rcheck/00install.out’ for details.
    ```

# CeTF

<details>

* Version: 1.14.0
* GitHub: NA
* Source code: https://github.com/cran/CeTF
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 230

Run `revdepcheck::revdep_details(, "CeTF")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        data   1.5Mb
        doc    3.1Mb
    ```

# CINNA

<details>

* Version: 1.2.2
* GitHub: NA
* Source code: https://github.com/cran/CINNA
* Date/Publication: 2023-08-08 16:40:02 UTC
* Number of recursive dependencies: 149

Run `revdepcheck::revdep_details(, "CINNA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘circlize’ ‘utils’
      All declared Imports should be used.
    ```

# CluMSID

<details>

* Version: 1.18.0
* GitHub: https://github.com/tdepke/CluMSID
* Source code: https://github.com/cran/CluMSID
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 195

Run `revdepcheck::revdep_details(, "CluMSID")` for more info

</details>

## In both

*   checking whether package ‘CluMSID’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dbscan::as.dendrogram’ by ‘stats::as.dendrogram’ when loading ‘CluMSID’
    See ‘/ggally/revdep/checks.noindex/CluMSID/new/CluMSID.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.3Mb
      sub-directories of 1Mb or more:
        doc   5.6Mb
    ```

# clustrd

<details>

* Version: 1.4.0
* GitHub: NA
* Source code: https://github.com/cran/clustrd
* Date/Publication: 2022-07-16 23:20:06 UTC
* Number of recursive dependencies: 76

Run `revdepcheck::revdep_details(, "clustrd")` for more info

</details>

## In both

*   checking whether package ‘clustrd’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/clustrd/new/clustrd.Rcheck/00install.out’ for details.
    ```

# communication

<details>

* Version: 0.1
* GitHub: NA
* Source code: https://github.com/cran/communication
* Date/Publication: 2021-02-25 09:20:02 UTC
* Number of recursive dependencies: 108

Run `revdepcheck::revdep_details(, "communication")` for more info

</details>

## In both

*   checking C++ specification ... NOTE
    ```
      Specified C++11: please drop specification unless essential
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘GGally’ ‘RColorBrewer’ ‘abind’ ‘diagram’ ‘ggplot2’ ‘grid’ ‘gtable’
      ‘igraph’ ‘magrittr’ ‘plyr’ ‘purrr’ ‘reshape2’ ‘scales’ ‘useful’
      All declared Imports should be used.
    ```

# conserveR

<details>

* Version: 1.0.4
* GitHub: https://github.com/azizka/conserveR
* Source code: https://github.com/cran/conserveR
* Date/Publication: 2021-08-02 09:10:06 UTC
* Number of recursive dependencies: 64

Run `revdepcheck::revdep_details(, "conserveR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘scales’ ‘sna’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 244 marked UTF-8 strings
    ```

# corrViz

<details>

* Version: 0.1.0
* GitHub: NA
* Source code: https://github.com/cran/corrViz
* Date/Publication: 2023-06-30 11:40:07 UTC
* Number of recursive dependencies: 128

Run `revdepcheck::revdep_details(, "corrViz")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        doc   6.7Mb
    ```

# coveffectsplot

<details>

* Version: 1.0.5
* GitHub: https://github.com/smouksassi/coveffectsplot
* Source code: https://github.com/cran/coveffectsplot
* Date/Publication: 2024-01-18 14:10:02 UTC
* Number of recursive dependencies: 149

Run `revdepcheck::revdep_details(, "coveffectsplot")` for more info

</details>

## In both

*   checking whether package ‘coveffectsplot’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘data.table’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/coveffectsplot/new/coveffectsplot.Rcheck/00install.out’ for details.
    ```

# covsim

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/covsim
* Date/Publication: 2022-04-29 22:40:09 UTC
* Number of recursive dependencies: 107

Run `revdepcheck::revdep_details(, "covsim")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘gsl’
      All declared Imports should be used.
    ```

# cyanoFilter

<details>

* Version: 1.10.0
* GitHub: https://github.com/fomotis/cyanoFilter
* Source code: https://github.com/cran/cyanoFilter
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 146

Run `revdepcheck::revdep_details(, "cyanoFilter")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘flowClust’

    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# descriptio

<details>

* Version: 1.2
* GitHub: https://github.com/nicolas-robette/descriptio
* Source code: https://github.com/cran/descriptio
* Date/Publication: 2023-10-05 12:50:02 UTC
* Number of recursive dependencies: 98

Run `revdepcheck::revdep_details(, "descriptio")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘FactoMineR’, ‘vcd’
    ```

# fect

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/fect
* Date/Publication: 2022-10-14 09:52:32 UTC
* Number of recursive dependencies: 81

Run `revdepcheck::revdep_details(, "fect")` for more info

</details>

## In both

*   checking C++ specification ... NOTE
    ```
      Specified C++11: please drop specification unless essential
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘panelView’
      All declared Imports should be used.
    ```

# fic

<details>

* Version: 1.0.0
* GitHub: https://github.com/chjackson/fic
* Source code: https://github.com/cran/fic
* Date/Publication: 2019-04-13 08:32:39 UTC
* Number of recursive dependencies: 119

Run `revdepcheck::revdep_details(, "fic")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘numDeriv’
      All declared Imports should be used.
    ```

# finalfit

<details>

* Version: 1.0.7
* GitHub: https://github.com/ewenharrison/finalfit
* Source code: https://github.com/cran/finalfit
* Date/Publication: 2023-11-16 17:40:02 UTC
* Number of recursive dependencies: 172

Run `revdepcheck::revdep_details(, "finalfit")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        doc   4.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyselect’
      All declared Imports should be used.
    ```

# fingerPro

<details>

* Version: 1.1
* GitHub: https://github.com/NA/NA
* Source code: https://github.com/cran/fingerPro
* Date/Publication: 2018-08-28 10:04:54 UTC
* Number of recursive dependencies: 161

Run `revdepcheck::revdep_details(, "fingerPro")` for more info

</details>

## In both

*   checking whether package ‘fingerPro’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/ggally/revdep/checks.noindex/fingerPro/new/fingerPro.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘fingerPro’ ...
** package ‘fingerPro’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
using C++ compiler: ‘Apple clang version 15.0.0 (clang-1500.1.0.2.5)’
using SDK: ‘MacOSX14.2.sdk’
clang++ -arch arm64 -std=gnu++17 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/ggally/revdep/library.noindex/fingerPro/Rcpp/include' -I'/ggally/revdep/library.noindex/fingerPro/RcppGSL/include' -I'/ggally/revdep/library.noindex/fingerPro/RcppProgress/include' -I/opt/R/arm64/include -I/usr/local/include -Xclang -fopenmp    -fPIC  -falign-functions=64 -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
clang++ -arch arm64 -std=gnu++17 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/ggally/revdep/library.noindex/fingerPro/Rcpp/include' -I'/ggally/revdep/library.noindex/fingerPro/RcppGSL/include' -I'/ggally/revdep/library.noindex/fingerPro/RcppProgress/include' -I/opt/R/arm64/include -I/usr/local/include -Xclang -fopenmp    -fPIC  -falign-functions=64 -Wall -g -O2  -c fingerprinting.cpp -o fingerprinting.o
In file included from RcppExports.cpp:4:
In file included from /ggally/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSL.h:25:
...
make: *** Waiting for unfinished jobs....
In file included from fingerprinting.cpp:5:
In file included from /ggally//ggally/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSL.h:25:
/ggally/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSLForward.h:25:10: fatal error: 'gsl/gsl_vector.h' file not found
#include <gsl/gsl_vector.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [fingerprinting.o] Error 1
ERROR: compilation failed for package ‘fingerPro’
* removing ‘/ggally/revdep/checks.noindex/fingerPro/new/fingerPro.Rcheck/fingerPro’


```
### CRAN

```
* installing *source* package ‘fingerPro’ ...
** package ‘fingerPro’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
using C++ compiler: ‘Apple clang version 15.0.0 (clang-1500.1.0.2.5)’
using SDK: ‘MacOSX14.2.sdk’
clang++ -arch arm64 -std=gnu++17 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/ggally//ggally/revdep/library.noindex/fingerPro/Rcpp/include' -I'/ggally/revdep/library.noindex/fingerPro/RcppGSL/include' -I'/ggally//ggally/revdep/library.noindex/fingerPro/RcppProgress/include' -I/opt/R/arm64/include -I/usr/local/include -Xclang -fopenmp    -fPIC  -falign-functions=64 -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
clang++ -arch arm64 -std=gnu++17 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/ggally/revdep/library.noindex/fingerPro/Rcpp/include' -I'/ggally/revdep/library.noindex/fingerPro/RcppGSL/include' -I'/ggally/revdep/library.noindex/fingerPro/RcppProgress/include' -I/opt/R/arm64/include -I/usr/local/include -Xclang -fopenmp    -fPIC  -falign-functions=64 -Wall -g -O2  -c fingerprinting.cpp -o fingerprinting.o
In file included from RcppExports.cpp:4:
In file included from /ggally/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSL.h:25:
...
make: *** Waiting for unfinished jobs....
In file included from fingerprinting.cpp:5:
In file included from /ggally/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSL.h:25:
/ggally/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSLForward.h:25:10: fatal error: 'gsl/gsl_vector.h' file not found
#include <gsl/gsl_vector.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [fingerprinting.o] Error 1
ERROR: compilation failed for package ‘fingerPro’
* removing ‘/ggally/revdep/checks.noindex/fingerPro/old/fingerPro.Rcheck/fingerPro’


```
# FPDclustering

<details>

* Version: 2.3.1
* GitHub: NA
* Source code: https://github.com/cran/FPDclustering
* Date/Publication: 2024-01-30 00:10:06 UTC
* Number of recursive dependencies: 104

Run `revdepcheck::revdep_details(, "FPDclustering")` for more info

</details>

## In both

*   checking whether package ‘FPDclustering’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘mvtnorm’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/FPDclustering/new/FPDclustering.Rcheck/00install.out’ for details.
    ```

# fpp2

<details>

* Version: 2.5
* GitHub: https://github.com/robjhyndman/fpp2-package
* Source code: https://github.com/cran/fpp2
* Date/Publication: 2023-02-19 07:30:02 UTC
* Number of recursive dependencies: 83

Run `revdepcheck::revdep_details(, "fpp2")` for more info

</details>

## In both

*   checking whether package ‘fpp2’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/fpp2/new/fpp2.Rcheck/00install.out’ for details.
    ```

# freqparcoord

<details>

* Version: 1.0.1
* GitHub: NA
* Source code: https://github.com/cran/freqparcoord
* Date/Publication: 2016-01-17 10:59:33
* Number of recursive dependencies: 60

Run `revdepcheck::revdep_details(, "freqparcoord")` for more info

</details>

## In both

*   checking whether package ‘freqparcoord’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ggplot2’ was built under R version 4.3.1
      Warning: package ‘FNN’ was built under R version 4.3.1
      Warning: package ‘mvtnorm’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/freqparcoord/new/freqparcoord.Rcheck/00install.out’ for details.
    ```

# gatom

<details>

* Version: 1.0.0
* GitHub: https://github.com/ctlab/gatom
* Source code: https://github.com/cran/gatom
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 150

Run `revdepcheck::revdep_details(, "gatom")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘htmlwidgets:::createWidgetId’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .makeEdgeTable: no visible binding for global variable ‘gene’
    .makeEdgeTable: no visible binding for global variable ‘ID’
    .makeEdgeTable: no visible binding for global variable ‘pval’
    .makeEdgeTable: no visible binding for global variable ‘enzyme’
    .makeEdgeTable: no visible binding for global variable ‘reaction’
    .makeEdgeTable: no visible binding for global variable ‘origin’
    .makeEdgeTable: no visible binding for global variable ‘atom.x’
    .makeEdgeTable: no visible binding for global variable ‘atom.y’
    .makeVertexTable: no visible binding for global variable ‘ID’
    .makeVertexTable: no visible binding for global variable ‘pval’
    ...
    Undefined global functions or variables:
      ID PATHNAME aggregate atom atom.x atom.y base_metabolite dev.off
      enzyme from gene keys metabolite metabolite.x metabolite.y na.omit
      name organism origin pdf pval reaction read.table rgb setNames signal
      signalRank species symbol tail to
    Consider adding
      importFrom("grDevices", "dev.off", "pdf", "rgb")
      importFrom("stats", "aggregate", "na.omit", "setNames")
      importFrom("utils", "read.table", "tail")
    to your NAMESPACE file.
    ```

# GENESIS

<details>

* Version: 2.32.0
* GitHub: https://github.com/UW-GAC/GENESIS
* Source code: https://github.com/cran/GENESIS
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 195

Run `revdepcheck::revdep_details(, "GENESIS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        doc       3.2Mb
        extdata   3.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘survey:::saddle’
      See the note in ?`:::` about the use of this operator.
    ```

# GeomxTools

<details>

* Version: 3.6.2
* GitHub: NA
* Source code: https://github.com/cran/GeomxTools
* Date/Publication: 2024-02-09
* Number of recursive dependencies: 229

Run `revdepcheck::revdep_details(, "GeomxTools")` for more info

</details>

## In both

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ Failed tests ════════════════════════════════════════════════════════════════
      ── Failure ('test_GeoMxSet_probeCollapse.R:61:3'): test that collapsed probe count is equal to the geomean of probes for each target ──
      sum(matches) == nrow(aggTestData@assayData$exprs) * numDCC is not TRUE

      `actual`:   FALSE
      `expected`: TRUE
      ── Failure ('test_GeoMxSet_probeCollapse.R:100:3'): test that the geomean and geosd of negatives is correct ──
      sum(matches) == length(unique(PKC$Module)) * numDCC * 2 is not TRUE

      `actual`:   FALSE
      `expected`: TRUE

      [ FAIL 2 | WARN 16 | SKIP 0 | PASS 288 ]
      Error: Test failures
      Execution halted
    ```

*   checking whether package ‘GeomxTools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Biobase’ was built under R version 4.3.1
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
      Warning: package ‘NanoStringNCTools’ was built under R version 4.3.1
      Warning: package ‘S4Vectors’ was built under R version 4.3.2
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/GeomxTools/new/GeomxTools.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 24.5Mb
      sub-directories of 1Mb or more:
        doc       5.2Mb
        extdata  18.4Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    License components which are templates and need '+ file LICENSE':
      MIT
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    aggregateCounts: no visible binding for global variable ‘TargetName’
    hkNorm: no visible binding for global variable ‘TargetName’
    mixedModelDE: multiple local function definitions for ‘deFunc’ with
      different formal arguments
    qgrubbs: no visible global function definition for ‘qtable’
    readPKCFile: no visible binding for global variable ‘RTS_ID’
    readPKCFile: no visible binding for global variable ‘Module’
    setGrubbsFlags: no visible binding for global variable ‘TargetName’
    setGrubbsFlags : <anonymous>: no visible binding for global variable
      ‘Module’
    ...
      variable ‘Module’
    setSignatureFuncs<-,NanoStringGeoMxSet-character: no visible global
      function definition for ‘setSigFuncs<-’
    signatureScores,NanoStringGeoMxSet: no visible global function
      definition for ‘assayDataElement2’
    signatureScores,NanoStringGeoMxSet: no visible global function
      definition for ‘.sigCalc’
    Undefined global functions or variables:
      .sigCalc Module RTS_ID TargetName analtye assayDataElement2 qtable
      setSigFuncs<-
    ```

# gfilmm

<details>

* Version: 2.0.5
* GitHub: https://github.com/stla/gfilmm
* Source code: https://github.com/cran/gfilmm
* Date/Publication: 2022-07-11 23:20:06 UTC
* Number of recursive dependencies: 134

Run `revdepcheck::revdep_details(, "gfilmm")` for more info

</details>

## In both

*   checking C++ specification ... NOTE
    ```
      Specified C++11: please drop specification unless essential
    ```

# ggbio

<details>

* Version: 1.50.0
* GitHub: https://github.com/lawremi/ggbio
* Source code: https://github.com/cran/ggbio
* Date/Publication: 2023-11-01
* Number of recursive dependencies: 186

Run `revdepcheck::revdep_details(, "ggbio")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘ggbio-Ex.R’ failed
    The error most likely occurred in:

    > ### Name: autoplot
    > ### Title: Generic autoplot function
    > ### Aliases: autoplot autoplot,GRanges-method autoplot,GRangesList-method
    > ###   autoplot,IRanges-method autoplot,Seqinfo-method
    > ###   autoplot,BSgenome-method autoplot,GAlignments-method
    > ###   autoplot,BamFile-method autoplot,BamFileList-method
    > ###   autoplot,TxDbOREnsDb-method autoplot,character-method
    ...
    > ## End(Not run)
    >
    >
    > ###################################################
    > ### code chunk number 56: bs-v
    > ###################################################
    > library(BSgenome.Hsapiens.UCSC.hg19)
    Error in library(BSgenome.Hsapiens.UCSC.hg19) :
      there is no package called 'BSgenome.Hsapiens.UCSC.hg19'
    Execution halted
    ```

*   checking whether package ‘ggbio’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/ggbio/new/ggbio.Rcheck/00install.out’ for details.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented S4 methods:
      generic '+' and siglist 'Tracks,ANY'
      generic '+' and siglist 'Tracks,Tracks'
      generic '+' and siglist 'Tracks,cartesian'
      generic '+' and siglist 'Tracks,position_c'
      generic '+' and siglist 'Tracks,theme'
      generic '+' and siglist 'Tracks,zoom'
    All user-level objects in a package (including S4 classes and methods)
    should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'geom_alignment':
    \S4method{geom_alignment}{GRanges}
      Code: function(data, ..., xlab, ylab, main, facets = NULL, stat =
                     c("stepping", "identity"), range.geom = c("rect",
                     "arrowrect"), gap.geom = c("chevron", "arrow",
                     "segment"), rect.height = NULL, group.selfish = TRUE)
      Docs: function(data, ..., xlab, ylab, main, facets = NULL, stat =
                     c("stepping", "identity"), range.geom = c("rect",
                     "arrowrect"), gap.geom = c("chevron", "arrow",
                     "segment"), rect.height = NULL, group.selfish = TRUE,
    ...
                     NULL, scale = 1, width = NA, height = NA, units =
                     c("in", "cm", "mm"), dpi = 300, limitsize = TRUE, ...)
      Docs: function(filename, plot = last_plot(), device =
                     default_device(filename), path = NULL, scale = 1,
                     width = par("din")[1], height = par("din")[2], units =
                     c("in", "cm", "mm"), dpi = 300, limitsize = TRUE, ...)
      Mismatches in argument default values:
        Name: 'device' Code: NULL Docs: default_device(filename)
        Name: 'width' Code: NA Docs: par("din")[1]
        Name: 'height' Code: NA Docs: par("din")[2]
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘BSgenome.Hsapiens.UCSC.hg19’
    ```

*   checking dependencies in R code ... NOTE
    ```
    ':::' call which should be '::': 'ggplot2:::set_last_plot'
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      'S4Vectors:::top_prenv' 'ggplot2:::add_ggplot' 'ggplot2:::cunion'
      'ggplot2:::rescale01'
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .combineNames: no visible binding for global variable
      '.layout_circle.stats'
    Ideogram: no visible global function definition for 'data'
    Ideogram: no visible binding for global variable 'ideoCyto'
    getNR: no visible global function definition for 'se'
    getNR: no visible global function definition for 'indexProbesProcessed'
    getNR: no visible global function definition for 'coefs'
    scale_x_sequnit: no visible binding for global variable '.x'
    autoplot,ExpressionSet: no visible binding for global variable
      'variable'
    ...
    stat_mismatch,GRanges: no visible binding for global variable 'sts'
    stat_mismatch,GRanges: no visible binding for global variable 'eds'
    stat_mismatch,GRanges: no visible binding for global variable 'read'
    Undefined global functions or variables:
      .fragLength .layout_circle.stats .x coefs data eds fl gieStain
      ideoCyto indexProbesProcessed midpoint mt name read se stepping sts
      value variable x xend y2 yend2
    Consider adding
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

*   checking Rd files ... NOTE
    ```
    checkRd: (-1) autoplot-method.Rd:445: Escaped LaTeX specials: \~
    ```

# ggmcmc

<details>

* Version: 1.5.1.1
* GitHub: https://github.com/xfim/ggmcmc
* Source code: https://github.com/cran/ggmcmc
* Date/Publication: 2021-02-10 10:50:10 UTC
* Number of recursive dependencies: 86

Run `revdepcheck::revdep_details(, "ggmcmc")` for more info

</details>

## In both

*   checking whether package ‘ggmcmc’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘dplyr’ was built under R version 4.3.1
      Warning: package ‘tidyr’ was built under R version 4.3.1
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/ggmcmc/new/ggmcmc.Rcheck/00install.out’ for details.
    ```

# GGMncv

<details>

* Version: 2.1.1
* GitHub: https://github.com/donaldRwilliams/GGMncv
* Source code: https://github.com/cran/GGMncv
* Date/Publication: 2021-12-15 07:40:28 UTC
* Number of recursive dependencies: 184

Run `revdepcheck::revdep_details(, "GGMncv")` for more info

</details>

## In both

*   checking C++ specification ... NOTE
    ```
      Specified C++11: please drop specification unless essential
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rdpack’ ‘mathjaxr’
      All declared Imports should be used.
    ```

*   checking Rd files ... NOTE
    ```
    checkRd: (-1) boot_eip.Rd:50: Escaped LaTeX specials: \&
    checkRd: (-1) constrained.Rd:93: Escaped LaTeX specials: \&
    ```

# GGMnonreg

<details>

* Version: 1.0.0
* GitHub: NA
* Source code: https://github.com/cran/GGMnonreg
* Date/Publication: 2021-04-08 11:30:06 UTC
* Number of recursive dependencies: 133

Run `revdepcheck::revdep_details(, "GGMnonreg")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘Rdpack’
      All declared Imports should be used.
    ```

*   checking Rd files ... NOTE
    ```
    checkRd: (-1) constrained.Rd:37: Escaped LaTeX specials: \&
    ```

# GGPA

<details>

* Version: 1.14.0
* GitHub: https://github.com/dongjunchung/GGPA
* Source code: https://github.com/cran/GGPA
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 88

Run `revdepcheck::revdep_details(, "GGPA")` for more info

</details>

## In both

*   checking whether package ‘GGPA’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      3_Param.cpp:74:12: warning: use of bitwise '&' with boolean operands [-Wbitwise-instead-of-logical]
      Warning: package ‘ggplot2’ was built under R version 4.3.1
      Warning: package ‘network’ was built under R version 4.3.1
      Warning: package ‘sna’ was built under R version 4.3.1
      Warning: package ‘scales’ was built under R version 4.3.1
      Warning: package ‘matrixStats’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/GGPA/new/GGPA.Rcheck/00install.out’ for details.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘network’ ‘scales’ ‘sna’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘GGPA/libs/GGPA.so’:
      Found ‘__ZNSt3__14coutE’, possibly from ‘std::cout’ (C++)
        Object: ‘3_Param.o’

    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs nor [v]sprintf.

    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# ggpcp

<details>

* Version: 0.2.0
* GitHub: https://github.com/heike/ggpcp
* Source code: https://github.com/cran/ggpcp
* Date/Publication: 2022-11-28 09:30:08 UTC
* Number of recursive dependencies: 93

Run `revdepcheck::revdep_details(, "ggpcp")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ggdensity’
    ```

# ggpicrust2

<details>

* Version: 1.7.3
* GitHub: https://github.com/cafferychen777/ggpicrust2
* Source code: https://github.com/cran/ggpicrust2
* Date/Publication: 2023-11-08 16:10:02 UTC
* Number of recursive dependencies: 249

Run `revdepcheck::revdep_details(, "ggpicrust2")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘MicrobiomeStat’

    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ggPMX

<details>

* Version: 1.2.11
* GitHub: https://github.com/ggPMXdevelopment/ggPMX
* Source code: https://github.com/cran/ggPMX
* Date/Publication: 2023-11-30 16:10:06 UTC
* Number of recursive dependencies: 181

Run `revdepcheck::revdep_details(, "ggPMX")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘lixoftConnectors’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.9Mb
      sub-directories of 1Mb or more:
        doc        1.1Mb
        help       2.4Mb
        testdata   4.8Mb
    ```

# ggquickeda

<details>

* Version: 0.3.1
* GitHub: https://github.com/smouksassi/ggquickeda
* Source code: https://github.com/cran/ggquickeda
* Date/Publication: 2024-01-15 10:20:02 UTC
* Number of recursive dependencies: 188

Run `revdepcheck::revdep_details(, "ggquickeda")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘Formula’ ‘GGally’ ‘Hmisc’ ‘RPostgres’ ‘colourpicker’ ‘ggpmisc’
      ‘ggpubr’ ‘glue’ ‘gridExtra’ ‘markdown’ ‘patchwork’ ‘plotly’
      ‘quantreg’ ‘shinyFiles’ ‘shinyjqui’ ‘shinyjs’ ‘table1’ ‘zoo’
      All declared Imports should be used.
    ```

# glancedata

<details>

* Version: 1.0.1
* GitHub: NA
* Source code: https://github.com/cran/glancedata
* Date/Publication: 2019-11-22 23:10:05 UTC
* Number of recursive dependencies: 109

Run `revdepcheck::revdep_details(, "glancedata")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# gsynth

<details>

* Version: 1.2.1
* GitHub: NA
* Source code: https://github.com/cran/gsynth
* Date/Publication: 2021-08-06 07:50:05 UTC
* Number of recursive dependencies: 77

Run `revdepcheck::revdep_details(, "gsynth")` for more info

</details>

## In both

*   checking C++ specification ... NOTE
    ```
      Specified C++11: please drop specification unless essential
    ```

# ibawds

<details>

* Version: 0.5.0
* GitHub: https://github.com/stibu81/ibawds
* Source code: https://github.com/cran/ibawds
* Date/Publication: 2022-06-17 08:20:07 UTC
* Number of recursive dependencies: 199

Run `revdepcheck::revdep_details(, "ibawds")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# ICSClust

<details>

* Version: 0.1.0
* GitHub: https://github.com/AuroreAA/ICSClust
* Source code: https://github.com/cran/ICSClust
* Date/Publication: 2023-09-21 13:20:02 UTC
* Number of recursive dependencies: 137

Run `revdepcheck::revdep_details(, "ICSClust")` for more info

</details>

## In both

*   checking whether package ‘ICSClust’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ICS’ was built under R version 4.3.1
      Warning: package ‘mvtnorm’ was built under R version 4.3.1
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/ICSClust/new/ICSClust.Rcheck/00install.out’ for details.
    ```

# ICtest

<details>

* Version: 0.3-5
* GitHub: NA
* Source code: https://github.com/cran/ICtest
* Date/Publication: 2022-05-18 07:30:29 UTC
* Number of recursive dependencies: 97

Run `revdepcheck::revdep_details(, "ICtest")` for more info

</details>

## In both

*   checking whether package ‘ICtest’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘JADE’ was built under R version 4.3.1
      Warning: package ‘ICS’ was built under R version 4.3.1
      Warning: package ‘mvtnorm’ was built under R version 4.3.1
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/ICtest/new/ICtest.Rcheck/00install.out’ for details.
    ```

# InflectSSP

<details>

* Version: 1.6
* GitHub: NA
* Source code: https://github.com/cran/InflectSSP
* Date/Publication: 2023-04-19 08:00:03 UTC
* Number of recursive dependencies: 97

Run `revdepcheck::revdep_details(, "InflectSSP")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘svglite’
      All declared Imports should be used.
    ```

# isomiRs

<details>

* Version: 1.30.0
* GitHub: https://github.com/lpantano/isomiRs
* Source code: https://github.com/cran/isomiRs
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 173

Run `revdepcheck::revdep_details(, "isomiRs")` for more info

</details>

## In both

*   checking whether package ‘isomiRs’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘SummarizedExperiment’ was built under R version 4.3.2
      Warning: package ‘MatrixGenerics’ was built under R version 4.3.1
      Warning: package ‘matrixStats’ was built under R version 4.3.1
      Warning: package ‘GenomicRanges’ was built under R version 4.3.1
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
      Warning: package ‘S4Vectors’ was built under R version 4.3.2
      Warning: package ‘IRanges’ was built under R version 4.3.1
      Warning: package ‘GenomeInfoDb’ was built under R version 4.3.2
      Warning: package ‘Biobase’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/isomiRs/new/isomiRs.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .clean_noise: no visible binding for global variable ‘total’
    .clean_noise: no visible binding for global variable ‘hits’
    .remove_gt_n_changes: no visible binding for global variable ‘changes’
    isoAnnotate: no visible binding for global variable ‘pct’
    isoCounts: no visible global function definition for ‘as.tibble’
    isoPlot: no visible binding for global variable ‘iso_sample’
    isoPlotPosition: no visible binding for global variable ‘iso_sample’
    mirna2targetscan: no visible binding for global variable
      ‘targetscan.Hs.egMIRNA’
    mirna2targetscan: no visible binding for global variable
    ...
    mirna2targetscan: no visible binding for global variable
      ‘targetscan.Mm.egTARGETS’
    mirna2targetscan: no visible binding for global variable
      ‘targetscan.Mm.egTARGETSFULL’
    Undefined global functions or variables:
      as.tibble changes hits iso_sample pct targetscan.Hs.egMIRBASE2FAMILY
      targetscan.Hs.egMIRNA targetscan.Hs.egTARGETS
      targetscan.Hs.egTARGETSFULL targetscan.Mm.egMIRBASE2FAMILY
      targetscan.Mm.egMIRNA targetscan.Mm.egTARGETS
      targetscan.Mm.egTARGETSFULL total
    ```

# ITNr

<details>

* Version: 0.7.0
* GitHub: NA
* Source code: https://github.com/cran/ITNr
* Date/Publication: 2023-03-31 14:10:11 UTC
* Number of recursive dependencies: 95

Run `revdepcheck::revdep_details(, "ITNr")` for more info

</details>

## In both

*   checking whether package ‘ITNr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘network’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/ITNr/new/ITNr.Rcheck/00install.out’ for details.
    ```

# jmv

<details>

* Version: 2.4.11
* GitHub: https://github.com/jamovi/jmv
* Source code: https://github.com/cran/jmv
* Date/Publication: 2023-10-12 13:20:02 UTC
* Number of recursive dependencies: 192

Run `revdepcheck::revdep_details(, "jmv")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.8Mb
      sub-directories of 1Mb or more:
        R      4.0Mb
        i18n   6.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BayesFactor’ ‘GGally’ ‘GPArotation’ ‘MASS’ ‘PMCMR’ ‘R6’ ‘ROCR’
      ‘afex’ ‘car’ ‘dplyr’ ‘emmeans’ ‘ggrepel’ ‘ggridges’ ‘lavaan’
      ‘multcomp’ ‘mvnormtest’ ‘nnet’ ‘psych’ ‘vcd’ ‘vcdExtra’
      All declared Imports should be used.
    ```

# klovan

<details>

* Version: 0.1.0
* GitHub: NA
* Source code: https://github.com/cran/klovan
* Date/Publication: 2024-02-01 10:20:02 UTC
* Number of recursive dependencies: 136

Run `revdepcheck::revdep_details(, "klovan")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RGeostats’
    ```

# loon.ggplot

<details>

* Version: 1.3.3
* GitHub: https://github.com/great-northern-diver/loon.ggplot
* Source code: https://github.com/cran/loon.ggplot
* Date/Publication: 2022-11-12 22:30:02 UTC
* Number of recursive dependencies: 117

Run `revdepcheck::revdep_details(, "loon.ggplot")` for more info

</details>

## In both

*   checking whether package ‘loon.ggplot’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/ggally/revdep/checks.noindex/loon.ggplot/new/loon.ggplot.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘loon.ggplot’ ...
** package ‘loon.ggplot’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘loon’:
 .onLoad failed in loadNamespace() for 'loon', details:
  call: structure(.External(.C_dotTcl, ...), class = "tclObj")
  error: [tcl] couldn't connect to display "".
In addition: Warning message:
In fun(libname, pkgname) : couldn't connect to display ""
Execution halted
ERROR: lazy loading failed for package ‘loon.ggplot’
* removing ‘/ggally/revdep/checks.noindex/loon.ggplot/new/loon.ggplot.Rcheck/loon.ggplot’


```
### CRAN

```
* installing *source* package ‘loon.ggplot’ ...
** package ‘loon.ggplot’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘loon’:
 .onLoad failed in loadNamespace() for 'loon', details:
  call: structure(.External(.C_dotTcl, ...), class = "tclObj")
  error: [tcl] couldn't connect to display "".
In addition: Warning message:
In fun(libname, pkgname) : couldn't connect to display ""
Execution halted
ERROR: lazy loading failed for package ‘loon.ggplot’
* removing ‘/ggally/revdep/checks.noindex/loon.ggplot/old/loon.ggplot.Rcheck/loon.ggplot’


```
# MAINT.Data

<details>

* Version: 2.7.1
* GitHub: NA
* Source code: https://github.com/cran/MAINT.Data
* Date/Publication: 2023-04-04 08:10:02 UTC
* Number of recursive dependencies: 90

Run `revdepcheck::revdep_details(, "MAINT.Data")` for more info

</details>

## In both

*   checking whether package ‘MAINT.Data’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rcpp’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/MAINT.Data/new/MAINT.Data.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        R      2.1Mb
        data   2.3Mb
    ```

# MAST

<details>

* Version: 1.28.0
* GitHub: https://github.com/RGLab/MAST
* Source code: https://github.com/cran/MAST
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 226

Run `revdepcheck::revdep_details(, "MAST")` for more info

</details>

## In both

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ Skipped tests (5) ═══════════════════════════════════════════════════════════
      • empty test (5): 'test-GSEA-by-boot.R:99:1', 'test-SingleCellAssay.R:298:1',
        'test-lmWrapper-glmer.R:8:1', 'test-sparsematrix.R:18:1',
        'test-sparsematrix.R:26:1'

      ══ Failed tests ════════════════════════════════════════════════════════════════
      ── Failure ('test-lrtest.R:25:5'): LRT and zlm are equivalent ──────────────────
      all(lrout$lrstat[smallDOF] >= lrout2[smallDOF, 3, 1]) is not TRUE

      `actual`:   FALSE
      `expected`: TRUE

      [ FAIL 1 | WARN 8 | SKIP 5 | PASS 292 ]
      Error: Test failures
      Execution halted
    ```

*   checking whether package ‘MAST’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘SingleCellExperiment’ was built under R version 4.3.2
      Warning: package ‘SummarizedExperiment’ was built under R version 4.3.2
      Warning: package ‘MatrixGenerics’ was built under R version 4.3.1
      Warning: package ‘matrixStats’ was built under R version 4.3.1
      Warning: package ‘GenomicRanges’ was built under R version 4.3.1
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
      Warning: package ‘S4Vectors’ was built under R version 4.3.2
      Warning: package ‘IRanges’ was built under R version 4.3.1
      Warning: package ‘GenomeInfoDb’ was built under R version 4.3.2
      Warning: package ‘Biobase’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/MAST/new/MAST.Rcheck/00install.out’ for details.
    ```

*   checking dependencies in R code ... WARNING
    ```
    Missing or unexported object: ‘gdata::xls2csv’
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.3Mb
      sub-directories of 1Mb or more:
        data   3.7Mb
        doc    5.3Mb
    ```

# MCbiclust

<details>

* Version: 1.26.0
* GitHub: NA
* Source code: https://github.com/cran/MCbiclust
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 214

Run `revdepcheck::revdep_details(, "MCbiclust")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.6Mb
      sub-directories of 1Mb or more:
        data   3.1Mb
        doc    5.6Mb
    ```

# mgcViz

<details>

* Version: 0.1.11
* GitHub: https://github.com/mfasiolo/mgcViz
* Source code: https://github.com/cran/mgcViz
* Date/Publication: 2023-10-06 10:50:02 UTC
* Number of recursive dependencies: 122

Run `revdepcheck::revdep_details(, "mgcViz")` for more info

</details>

## In both

*   checking whether package ‘mgcViz’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘mgcv’ was built under R version 4.3.1
      Warning: package ‘nlme’ was built under R version 4.3.1
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/mgcViz/new/mgcViz.Rcheck/00install.out’ for details.
    ```

# miaSim

<details>

* Version: 1.8.0
* GitHub: https://github.com/microbiome/miaSim
* Source code: https://github.com/cran/miaSim
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 206

Run `revdepcheck::revdep_details(, "miaSim")` for more info

</details>

## In both

*   checking whether package ‘miaSim’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘TreeSummarizedExperiment’ was built under R version 4.3.1
      Warning: package ‘SingleCellExperiment’ was built under R version 4.3.2
      Warning: package ‘SummarizedExperiment’ was built under R version 4.3.2
      Warning: package ‘MatrixGenerics’ was built under R version 4.3.1
      Warning: package ‘matrixStats’ was built under R version 4.3.1
      Warning: package ‘GenomicRanges’ was built under R version 4.3.1
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
      Warning: package ‘S4Vectors’ was built under R version 4.3.2
      Warning: package ‘IRanges’ was built under R version 4.3.1
      Warning: package ‘GenomeInfoDb’ was built under R version 4.3.2
      Warning: package ‘Biobase’ was built under R version 4.3.1
      Warning: package ‘Biostrings’ was built under R version 4.3.2
      Warning: package ‘XVector’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/miaSim/new/miaSim.Rcheck/00install.out’ for details.
    ```

# mlergm

<details>

* Version: 0.8
* GitHub: NA
* Source code: https://github.com/cran/mlergm
* Date/Publication: 2021-08-23 16:00:02 UTC
* Number of recursive dependencies: 94

Run `revdepcheck::revdep_details(, "mlergm")` for more info

</details>

## In both

*   checking whether package ‘mlergm’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ergm’ was built under R version 4.3.1
      Warning: package ‘network’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/mlergm/new/mlergm.Rcheck/00install.out’ for details.
    ```

# mlrMBO

<details>

* Version: 1.1.5.1
* GitHub: https://github.com/mlr-org/mlrMBO
* Source code: https://github.com/cran/mlrMBO
* Date/Publication: 2022-07-04 08:50:50 UTC
* Number of recursive dependencies: 142

Run `revdepcheck::revdep_details(, "mlrMBO")` for more info

</details>

## In both

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      (<https://mlr3.mlr-org.com>). Due to the focus on 'mlr3' there might be
      uncaught bugs meanwhile in {mlr} - please consider switching.
      Loading required package: smoof
      Loading required package: checkmate
      [ FAIL 1 | WARN 0 | SKIP 0 | PASS 59 ]

      ══ Failed tests ════════════════════════════════════════════════════════════════
      ── Failure ('test_mbo_km.R:47:3'): mbo works with km ───────────────────────────
      `suppressPackageStartupMessages(...)` produced warnings.

      [ FAIL 1 | WARN 0 | SKIP 0 | PASS 59 ]
      Error: Test failures
      In addition: Warning message:
      package 'checkmate' was built under R version 4.3.1
      Execution halted
    ```

*   checking whether package ‘mlrMBO’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘checkmate’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/mlrMBO/new/mlrMBO.Rcheck/00install.out’ for details.
    ```

# MOFA2

<details>

* Version: 1.12.1
* GitHub: https://github.com/bioFAM/MOFA2
* Source code: https://github.com/cran/MOFA2
* Date/Publication: 2024-01-05
* Number of recursive dependencies: 259

Run `revdepcheck::revdep_details(, "MOFA2")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘MOFA2-Ex.R’ failed
    The error most likely occurred in:

    > ### Name: plot_dimred
    > ### Title: Plot dimensionality reduction based on MOFA factors
    > ### Aliases: plot_dimred
    >
    > ### ** Examples
    >
    > # Using an existing trained model on simulated data
    > file <- system.file("extdata", "model.hdf5", package = "MOFA2")
    > model <- load_model(file)
    >
    > # Run UMAP
    > model <- run_umap(model)
    Error in irlba::irlba(L, nv = n, nu = 0, maxit = iters) :
      function 'as_cholmod_sparse' not provided by package 'Matrix'
    Calls: run_umap ... irlba_tsvd_normalized_laplacian_init -> irlba_spectral_tsvd
    Execution halted
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'create_mofa_from_Seurat':
    create_mofa_from_Seurat
      Code: function(seurat, groups = NULL, assays = NULL, layer = "data",
                     features = NULL, extract_metadata = FALSE)
      Docs: function(seurat, groups = NULL, assays = NULL, slot =
                     "scale.data", features = NULL, extract_metadata =
                     FALSE)
      Argument names in code not in docs:
        layer
      Argument names in docs not in code:
        slot
      Mismatches in argument names:
        Position: 4 Code: layer Docs: slot
    ```

*   checking whether package ‘MOFA2’ can be installed ... NOTE
    ```
    Found the following notes/warnings:
      Non-staged installation was used
    See ‘/ggally/revdep/checks.noindex/MOFA2/new/MOFA2.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.8Mb
      sub-directories of 1Mb or more:
        doc       4.7Mb
        extdata   2.0Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    .plot_factors_vs_cov_1d: no visible binding for global variable
      ‘value.factor’
    .plot_factors_vs_cov_1d: no visible binding for global variable
      ‘value_scaled’
    .plot_factors_vs_cov_1d: no visible binding for global variable
      ‘value.covariate’
    .plot_factors_vs_cov_1d: no visible binding for global variable ‘value’
    .plot_factors_vs_cov_2d: no visible binding for global variable
      ‘value.factor’
    .set_xax: no visible binding for global variable ‘value’
    ...
    plot_weights: no visible binding for global variable ‘value’
    plot_weights: no visible binding for global variable ‘.’
    summarise_factors: no visible binding for global variable ‘value’
    summarise_factors: no visible binding for global variable ‘level’
    summarise_factors: no visible binding for global variable ‘group’
    Undefined global functions or variables:
      . CreateDimReducObject E2 L1 L2 colData covariate covariate_value
      feature.statistic group group_label level ntotal pathway ptotal
      sample_id sharedness smoothness value value.covariate value.factor
      value.unaligned value.warped value_scaled variance view ymax ymin
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘PCGSE’
    ```

# NetworkChange

<details>

* Version: 0.8
* GitHub: https://github.com/jongheepark/NetworkChange
* Source code: https://github.com/cran/NetworkChange
* Date/Publication: 2022-03-04 07:30:02 UTC
* Number of recursive dependencies: 132

Run `revdepcheck::revdep_details(, "NetworkChange")` for more info

</details>

## In both

*   checking whether package ‘NetworkChange’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘MCMCpack’ was built under R version 4.3.1
      Warning: package ‘coda’ was built under R version 4.3.1
      Warning: package ‘MASS’ was built under R version 4.3.1
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/NetworkChange/new/NetworkChange.Rcheck/00install.out’ for details.
    ```

*   checking Rd files ... NOTE
    ```
    checkRd: (-1) NetworkChange.Rd:163: Escaped LaTeX specials: \&
    checkRd: (-1) NetworkChangeRobust.Rd:144: Escaped LaTeX specials: \&
    checkRd: (-1) NetworkStatic.Rd:142: Escaped LaTeX specials: \&
    ```

# NPflow

<details>

* Version: 0.13.5
* GitHub: https://github.com/sistm/NPflow
* Source code: https://github.com/cran/NPflow
* Date/Publication: 2024-01-13 10:00:02 UTC
* Number of recursive dependencies: 70

Run `revdepcheck::revdep_details(, "NPflow")` for more info

</details>

## In both

*   checking whether package ‘NPflow’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rcpp’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/NPflow/new/NPflow.Rcheck/00install.out’ for details.
    ```

# nzelect

<details>

* Version: 0.4.0
* GitHub: NA
* Source code: https://github.com/cran/nzelect
* Date/Publication: 2017-10-02 20:35:23 UTC
* Number of recursive dependencies: 95

Run `revdepcheck::revdep_details(, "nzelect")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 6409 marked UTF-8 strings
    ```

# pareg

<details>

* Version: 1.6.0
* GitHub: https://github.com/cbg-ethz/pareg
* Source code: https://github.com/cran/pareg
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 320

Run `revdepcheck::revdep_details(, "pareg")` for more info

</details>

## In both

*   R CMD check timed out


*   checking whether package ‘pareg’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘tensorflow’ was built under R version 4.3.1
    Found the following additional notes/warnings:
      Non-staged installation was used
    See ‘/ggally/revdep/checks.noindex/pareg/new/pareg.Rcheck/00install.out’ for details.
    ```

*   checking for portable file names ... NOTE
    ```
    Found the following non-portable file paths:
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_ablation_study/config.yaml
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_ablation_study/params.csv
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_dispersion_fitting/config.yaml
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_dispersion_fitting/params.csv
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_real_datasets/config.yaml
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_real_datasets/params.csv
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_regularization_effect/config.yaml
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_regularization_effect/params.csv
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_regularization_parameter/config.yaml
    ...
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_response_distribution/params.csv
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_similarity_measures/config.yaml
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_similarity_measures/params.csv
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/workflow/scripts/compare_rocauc_vs_loss.R
      pareg/inst/scripts/synthetic_benchmark/resources/multi_config_workflow/config_regularization_parameter

    Tarballs are only required to store paths of up to 100 bytes and cannot
    store those of more than 256 bytes, with restrictions including to 100
    bytes for the final component.
    See section ‘Package structure’ in the ‘Writing R Extensions’ manual.
    ```

# PAsso

<details>

* Version: 0.1.10
* GitHub: https://github.com/XiaoruiZhu/PAsso
* Source code: https://github.com/cran/PAsso
* Date/Publication: 2021-06-18 09:20:08 UTC
* Number of recursive dependencies: 180

Run `revdepcheck::revdep_details(, "PAsso")` for more info

</details>

## In both

*   checking whether package ‘PAsso’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ggplot2’ was built under R version 4.3.1
      Warning: package ‘dplyr’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/PAsso/new/PAsso.Rcheck/00install.out’ for details.
    ```

# PhosR

<details>

* Version: 1.12.0
* GitHub: NA
* Source code: https://github.com/cran/PhosR
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 193

Run `revdepcheck::revdep_details(, "PhosR")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.4Mb
      sub-directories of 1Mb or more:
        data   3.3Mb
        doc    2.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘stringi’
      All declared Imports should be used.
    Unexported objects imported by ':::' calls:
      ‘BiocGenerics:::replaceSlots’ ‘S4Vectors:::disableValidity’
      ‘SummarizedExperiment:::.SummarizedExperiment.charbound’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    abundPlot: no visible binding for global variable ‘.’
    Undefined global functions or variables:
      .
    ```

# Pi

<details>

* Version: 2.14.0
* GitHub: https://github.com/hfang-bristol/Pi
* Source code: https://github.com/cran/Pi
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 260

Run `revdepcheck::revdep_details(, "Pi")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘dnet’

    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# plotly

<details>

* Version: 4.10.4
* GitHub: https://github.com/plotly/plotly.R
* Source code: https://github.com/cran/plotly
* Date/Publication: 2024-01-13 22:40:02 UTC
* Number of recursive dependencies: 148

Run `revdepcheck::revdep_details(, "plotly")` for more info

</details>

## In both

*   checking whether package ‘plotly’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/plotly/new/plotly.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        R             1.0Mb
        htmlwidgets   4.0Mb
    ```

# PopGenReport

<details>

* Version: 3.1
* GitHub: https://github.com/green-striped-gecko/PopGenReport
* Source code: https://github.com/cran/PopGenReport
* Date/Publication: 2023-10-11 07:20:02 UTC
* Number of recursive dependencies: 141

Run `revdepcheck::revdep_details(, "PopGenReport")` for more info

</details>

## In both

*   checking whether package ‘PopGenReport’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘knitr’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/PopGenReport/new/PopGenReport.Rcheck/00install.out’ for details.
    ```

# protGear

<details>

* Version: 1.6.0
* GitHub: https://github.com/Keniajin/protGear
* Source code: https://github.com/cran/protGear
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 203

Run `revdepcheck::revdep_details(, "protGear")` for more info

</details>

## In both

*   checking whether package ‘protGear’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘dplyr’ was built under R version 4.3.1
      Warning: package ‘limma’ was built under R version 4.3.1
      Warning: package ‘vsn’ was built under R version 4.3.1
      Warning: package ‘Biobase’ was built under R version 4.3.1
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/protGear/new/protGear.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        doc       3.9Mb
        extdata   1.2Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    best_CV_estimation: no visible binding for global variable ‘x’
    best_CV_estimation: no visible binding for global variable ‘prop’
    best_CV_estimation: no visible binding for global variable ‘val’
    best_CV_estimation: no visible binding for global variable ‘meanX1_X2’
    best_CV_estimation: no visible binding for global variable ‘meanX2_X3’
    best_CV_estimation: no visible binding for global variable ‘xbar’
    best_CV_estimation: no visible binding for global variable ‘CVX’
    best_CV_estimation: no visible binding for global variable ‘meanX’
    best_CV_estimation: no visible binding for global variable ‘selected’
    best_CV_estimation: no visible binding for global variable ‘CVX2_X3’
    ...
    visualize_slide_2d: no visible binding for global variable ‘meanY’
    Undefined global functions or variables:
      .id Array BGMedian BG_Median Block CVX CVX1_X2 CVX1_X3 CVX2_X3 Column
      Description FBG_Median FMedian FMedianBG_correct MFI MFI_val Name Row
      TAG TAG_mfi TAG_name X Y antigen antigen_name best_CV buffers
      day_batches global_BGMedian index log_bg log_fb machines
      meanBest2_RLM meanX meanX1_X2 meanX1_X3 meanX2_X3 meanY mean_all_anti
      mean_mfi minimum_BGMedian prop rank_mean_all_anti sampleID sampleID2
      sample_array_ID sample_index sdX sdX1_X2 sdX1_X3 sdX2_X3 sd_mfi
      selected slide stdev_all_anti temp v1 v2 val value variable x xbar
    ```

# randomForestExplainer

<details>

* Version: 0.10.1
* GitHub: https://github.com/ModelOriented/randomForestExplainer
* Source code: https://github.com/cran/randomForestExplainer
* Date/Publication: 2020-07-11 20:30:02 UTC
* Number of recursive dependencies: 105

Run `revdepcheck::revdep_details(, "randomForestExplainer")` for more info

</details>

## In both

*   checking LazyData ... NOTE
    ```
      'LazyData' is specified without a 'data' directory
    ```

# rbioacc

<details>

* Version: 1.2-0
* GitHub: NA
* Source code: https://github.com/cran/rbioacc
* Date/Publication: 2023-09-21 09:30:06 UTC
* Number of recursive dependencies: 106

Run `revdepcheck::revdep_details(, "rbioacc")` for more info

</details>

## In both

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# RHMS

<details>

* Version: 1.7
* GitHub: NA
* Source code: https://github.com/cran/RHMS
* Date/Publication: 2021-09-27 15:50:02 UTC
* Number of recursive dependencies: 96

Run `revdepcheck::revdep_details(, "RHMS")` for more info

</details>

## In both

*   checking whether package ‘RHMS’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘network’ was built under R version 4.3.1
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/RHMS/new/RHMS.Rcheck/00install.out’ for details.
    ```

# ridgetorus

<details>

* Version: 1.0.2
* GitHub: https://github.com/egarpor/ridgetorus
* Source code: https://github.com/cran/ridgetorus
* Date/Publication: 2023-08-27 22:40:02 UTC
* Number of recursive dependencies: 142

Run `revdepcheck::revdep_details(, "ridgetorus")` for more info

</details>

## In both

*   checking whether package ‘ridgetorus’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rcpp’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/ridgetorus/new/ridgetorus.Rcheck/00install.out’ for details.
    ```

# robCompositions

<details>

* Version: 2.4.1
* GitHub: NA
* Source code: https://github.com/cran/robCompositions
* Date/Publication: 2023-08-25 15:30:06 UTC
* Number of recursive dependencies: 143

Run `revdepcheck::revdep_details(, "robCompositions")` for more info

</details>

## In both

*   checking whether package ‘robCompositions’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ggplot2’ was built under R version 4.3.1
      Warning: package ‘pls’ was built under R version 4.3.1
      Warning: package ‘data.table’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/robCompositions/new/robCompositions.Rcheck/00install.out’ for details.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# robustbase

<details>

* Version: 0.99-2
* GitHub: NA
* Source code: https://github.com/cran/robustbase
* Date/Publication: 2024-01-27 16:30:02 UTC
* Number of recursive dependencies: 78

Run `revdepcheck::revdep_details(, "robustbase")` for more info

</details>

## In both

*   checking whether package ‘robustbase’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/ggally/revdep/checks.noindex/robustbase/new/robustbase.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘robustX’
    ```

## Installation

### Devel

```
* installing *source* package ‘robustbase’ ...
** package ‘robustbase’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
using C compiler: ‘Apple clang version 15.0.0 (clang-1500.1.0.2.5)’
sh: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
using SDK: ‘MacOSX14.2.sdk’
clang -arch arm64 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/opt/R/arm64/include -I/usr/local/include -Xclang -fopenmp    -fPIC  -falign-functions=64 -Wall -g -O2  -c R-rng4ftn.c -o R-rng4ftn.o
/opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran  -fPIC  -Wall -g -O2  -c eigen.f -o eigen.o
make: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
...
make: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
/opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran  -fPIC  -Wall -g -O2  -c rfltsreg.f -o rfltsreg.o
make: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
make: *** [eigen.o] Error 1
make: *** Waiting for unfinished jobs....
make: *** [rfltsreg.o] Error 1
make: *** [rffastmcd.o] Error 1
make: *** [rf-common.o] Error 1
ERROR: compilation failed for package ‘robustbase’
* removing ‘/ggally/revdep/checks.noindex/robustbase/new/robustbase.Rcheck/robustbase’


```
### CRAN

```
* installing *source* package ‘robustbase’ ...
** package ‘robustbase’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
using C compiler: ‘Apple clang version 15.0.0 (clang-1500.1.0.2.5)’
sh: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
using SDK: ‘MacOSX14.2.sdk’
clang -arch arm64 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/opt/R/arm64/include -I/usr/local/include -Xclang -fopenmp    -fPIC  -falign-functions=64 -Wall -g -O2  -c R-rng4ftn.c -o R-rng4ftn.o
/opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran  -fPIC  -Wall -g -O2  -c eigen.f -o eigen.o
make: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
...
/opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran  -fPIC  -Wall -g -O2  -c rf-common.f -o rf-common.o
make: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
/opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran  -fPIC  -Wall -g -O2  -c rffastmcd.f -o rffastmcd.o
make: /opt/homebrew/Cellar/gcc/12.2.0/bin/gfortran: No such file or directory
make: *** [eigen.o] Error 1
make: *** Waiting for unfinished jobs....
make: *** [rf-common.o] Error 1
make: *** [rffastmcd.o] Error 1
ERROR: compilation failed for package ‘robustbase’
* removing ‘/ggally/revdep/checks.noindex/robustbase/old/robustbase.Rcheck/robustbase’


```
# rrr

<details>

* Version: 1.0.0
* GitHub: https://github.com/chrisaddy/rrr
* Source code: https://github.com/cran/rrr
* Date/Publication: 2016-12-09 15:15:55
* Number of recursive dependencies: 104

Run `revdepcheck::revdep_details(, "rrr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# rwty

<details>

* Version: 1.0.2
* GitHub: NA
* Source code: https://github.com/cran/rwty
* Date/Publication: 2019-02-15 15:10:03 UTC
* Number of recursive dependencies: 102

Run `revdepcheck::revdep_details(, "rwty")` for more info

</details>

## In both

*   checking whether package ‘rwty’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/rwty/new/rwty.Rcheck/00install.out’ for details.
    ```

# sageR

<details>

* Version: 0.6.1
* GitHub: https://github.com/fbertran/sageR
* Source code: https://github.com/cran/sageR
* Date/Publication: 2023-03-23 18:40:02 UTC
* Number of recursive dependencies: 249

Run `revdepcheck::revdep_details(, "sageR")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 120 marked UTF-8 strings
    ```

# scPipe

<details>

* Version: 2.2.0
* GitHub: https://github.com/LuyiTian/scPipe
* Source code: https://github.com/cran/scPipe
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 260

Run `revdepcheck::revdep_details(, "scPipe")` for more info

</details>

## In both

*   checking whether package ‘scPipe’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘SingleCellExperiment’ was built under R version 4.3.2
      Warning: package ‘SummarizedExperiment’ was built under R version 4.3.2
      Warning: package ‘MatrixGenerics’ was built under R version 4.3.1
      Warning: package ‘matrixStats’ was built under R version 4.3.1
      Warning: package ‘GenomicRanges’ was built under R version 4.3.1
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
      Warning: package ‘S4Vectors’ was built under R version 4.3.2
      Warning: package ‘IRanges’ was built under R version 4.3.1
      Warning: package ‘GenomeInfoDb’ was built under R version 4.3.2
      Warning: package ‘Biobase’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/scPipe/new/scPipe.Rcheck/00install.out’ for details.
    ```

*   checking C++ specification ... NOTE
    ```
      Specified C++11: please drop specification unless essential
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 17.0Mb
      sub-directories of 1Mb or more:
        doc       1.5Mb
        extdata  12.1Mb
        libs      2.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘flexmix’ ‘vctrs’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    sc_atac_feature_counting: warning in sc_atac_create_sce(input_folder =
      output_folder, organism = organism, sample = sample_name,
      feature_type = feature_type, pheno_data = pheno_data, report =
      create_report): partial argument match of 'sample' to 'sample_name'
    sc_aligning: no visible binding for global variable ‘Var1’
    sc_aligning: no visible binding for global variable ‘Freq’
    sc_atac_bam_tagging: no visible binding for global variable ‘.N’
    sc_atac_bam_tagging: no visible binding for global variable ‘count’
    sc_atac_bam_tagging: no visible binding for global variable ‘.SD’
    sc_atac_create_cell_qc_metrics: no visible binding for global variable
    ...
    sc_mae_plot_umap: no visible binding for global variable ‘UMAP2’
    Undefined global functions or variables:
      ..count.. .N .SD Freq UMAP1 UMAP2 Var1 barcode cluster count end
      experiments irlba log_cells_per_feature log_counts_per_cell
      log_counts_per_feature log_features_per_cell seqnames start value
      write.csv
    Consider adding
      importFrom("stats", "end", "start")
      importFrom("utils", "write.csv")
    to your NAMESPACE file.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# SeqSQC

<details>

* Version: 1.24.0
* GitHub: https://github.com/Liubuntu/SeqSQC
* Source code: https://github.com/cran/SeqSQC
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 144

Run `revdepcheck::revdep_details(, "SeqSQC")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rbokeh’

    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# SimBIID

<details>

* Version: 0.2.1
* GitHub: https://github.com/tjmckinley/SimBIID
* Source code: https://github.com/cran/SimBIID
* Date/Publication: 2021-02-04 05:20:02 UTC
* Number of recursive dependencies: 79

Run `revdepcheck::revdep_details(, "SimBIID")` for more info

</details>

## In both

*   checking C++ specification ... NOTE
    ```
      Specified C++11: please drop specification unless essential
    ```

# simmr

<details>

* Version: 0.5.1.216
* GitHub: https://github.com/andrewcparnell/simmr
* Source code: https://github.com/cran/simmr
* Date/Publication: 2023-10-27 14:20:02 UTC
* Number of recursive dependencies: 129

Run `revdepcheck::revdep_details(, "simmr")` for more info

</details>

## In both

*   checking whether package ‘simmr’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/ggally/revdep/checks.noindex/simmr/new/simmr.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘simmr’ ...
** package ‘simmr’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
using C++ compiler: ‘Apple clang version 15.0.0 (clang-1500.1.0.2.5)’
using SDK: ‘MacOSX14.2.sdk’
clang++ -arch arm64 -std=gnu++17 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/ggally/revdep/library.noindex/simmr/Rcpp/include' -I'/ggally/revdep/library.noindex/simmr/RcppArmadillo/include' -I'/ggally/revdep/library.noindex/simmr/RcppDist/include' -I/opt/R/arm64/include -I/usr/local/include -Xclang -fopenmp    -fPIC  -falign-functions=64 -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
clang++ -arch arm64 -std=gnu++17 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/ggally/revdep/library.noindex/simmr/Rcpp/include' -I'/ggally/revdep/library.noindex/simmr/RcppArmadillo/include' -I'/ggally/revdep/library.noindex/simmr/RcppDist/include' -I/opt/R/arm64/include -I/usr/local/include -Xclang -fopenmp    -fPIC  -falign-functions=64 -Wall -g -O2  -c run_VB.cpp -o run_VB.o
run_VB.cpp:6:21: warning: unused variable 'log2pi' [-Wunused-const-variable]
static double const log2pi = std::log(2.0 * M_PI);
...
  dlopen(/ggally/revdep/library.noindex/simmr/rjags/libs/rjags.so, 0x000A): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: <621A7307-C96A-3A25-BE2B-B9C2339518AE> /ggally/revdep/library.noindex/simmr/rjags/libs/rjags.so
  Reason: tried: '/usr/local/lib/libjags.4.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/lib/libjags.4.dylib' (no such file), '/usr/local/lib/libjags.4.dylib' (no such file), '/Library/Frameworks/R.framework/Resources/lib/libjags.4.dy
In addition: Warning messages:
1: package ‘R2jags’ was built under R version 4.3.1
2: package ‘rjags’ was built under R version 4.3.1
3: package ‘coda’ was built under R version 4.3.1
Execution halted
ERROR: lazy loading failed for package ‘simmr’
* removing ‘/ggally/revdep/checks.noindex/simmr/new/simmr.Rcheck/simmr’


```
### CRAN

```
* installing *source* package ‘simmr’ ...
** package ‘simmr’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
using C++ compiler: ‘Apple clang version 15.0.0 (clang-1500.1.0.2.5)’
using SDK: ‘MacOSX14.2.sdk’
clang++ -arch arm64 -std=gnu++17 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/ggally/revdep/library.noindex/simmr/Rcpp/include' -I'/ggally/revdep/library.noindex/simmr/RcppArmadillo/include' -I'/ggally/revdep/library.noindex/simmr/RcppDist/include' -I/opt/R/arm64/include -I/usr/local/include -Xclang -fopenmp    -fPIC  -falign-functions=64 -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
clang++ -arch arm64 -std=gnu++17 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/ggally/revdep/library.noindex/simmr/Rcpp/include' -I'/ggally/revdep/library.noindex/simmr/RcppArmadillo/include' -I'/ggally/revdep/library.noindex/simmr/RcppDist/include' -I/opt/R/arm64/include -I/usr/local/include -Xclang -fopenmp    -fPIC  -falign-functions=64 -Wall -g -O2  -c run_VB.cpp -o run_VB.o
run_VB.cpp:6:21: warning: unused variable 'log2pi' [-Wunused-const-variable]
static double const log2pi = std::log(2.0 * M_PI);
...
  dlopen(/ggally/revdep/library.noindex/simmr/rjags/libs/rjags.so, 0x000A): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: <621A7307-C96A-3A25-BE2B-B9C2339518AE> /ggally/revdep/library.noindex/simmr/rjags/libs/rjags.so
  Reason: tried: '/usr/local/lib/libjags.4.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/lib/libjags.4.dylib' (no such file), '/usr/local/lib/libjags.4.dylib' (no such file), '/Library/Frameworks/R.framework/Resources/lib/libjags.4.dy
In addition: Warning messages:
1: package ‘R2jags’ was built under R version 4.3.1
2: package ‘rjags’ was built under R version 4.3.1
3: package ‘coda’ was built under R version 4.3.1
Execution halted
ERROR: lazy loading failed for package ‘simmr’
* removing ‘/ggally/revdep/checks.noindex/simmr/old/simmr.Rcheck/simmr’


```
# SmartEDA

<details>

* Version: 0.3.10
* GitHub: https://github.com/daya6489/SmartEDA
* Source code: https://github.com/cran/SmartEDA
* Date/Publication: 2024-01-30 17:50:02 UTC
* Number of recursive dependencies: 108

Run `revdepcheck::revdep_details(, "SmartEDA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘qpdf’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘ggthemes’, ‘DataExplorer’
    ```

# SPOTMisc

<details>

* Version: 1.19.52
* GitHub: NA
* Source code: https://github.com/cran/SPOTMisc
* Date/Publication: 2022-09-05 15:10:02 UTC
* Number of recursive dependencies: 181

Run `revdepcheck::revdep_details(, "SPOTMisc")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        data   4.7Mb
    ```

# staRdom

<details>

* Version: 1.1.28
* GitHub: https://github.com/MatthiasPucher/staRdom
* Source code: https://github.com/cran/staRdom
* Date/Publication: 2023-06-16 08:20:02 UTC
* Number of recursive dependencies: 158

Run `revdepcheck::revdep_details(, "staRdom")` for more info

</details>

## In both

*   checking whether package ‘staRdom’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/staRdom/new/staRdom.Rcheck/00install.out’ for details.
    ```

# statVisual

<details>

* Version: 1.2.1
* GitHub: NA
* Source code: https://github.com/cran/statVisual
* Date/Publication: 2020-02-20 19:30:02 UTC
* Number of recursive dependencies: 194

Run `revdepcheck::revdep_details(, "statVisual")` for more info

</details>

## In both

*   checking whether package ‘statVisual’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Biobase’ was built under R version 4.3.1
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/statVisual/new/statVisual.Rcheck/00install.out’ for details.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gbm’ ‘ggfortify’ ‘tibble’ ‘tidyverse’
      All declared Imports should be used.
    ```

# survstan

<details>

* Version: 0.0.6.1
* GitHub: https://github.com/fndemarqui/survstan
* Source code: https://github.com/cran/survstan
* Date/Publication: 2024-02-05 15:00:02 UTC
* Number of recursive dependencies: 123

Run `revdepcheck::revdep_details(, "survstan")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RcppParallel’ ‘rstantools’
      All declared Imports should be used.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# systemPipeTools

<details>

* Version: 1.10.0
* GitHub: NA
* Source code: https://github.com/cran/systemPipeTools
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 164

Run `revdepcheck::revdep_details(, "systemPipeTools")` for more info

</details>

## In both

*   checking dependencies in R code ... WARNING
    ```
    Missing or unexported object: ‘BiocGenerics::rowSums’
    ```

# tidybulk

<details>

* Version: 1.14.3
* GitHub: https://github.com/stemangiola/tidybulk
* Source code: https://github.com/cran/tidybulk
* Date/Publication: 2023-12-04
* Number of recursive dependencies: 400

Run `revdepcheck::revdep_details(, "tidybulk")` for more info

</details>

## In both

*   checking whether package ‘tidybulk’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ttservice’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/tidybulk/new/tidybulk.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        data   2.3Mb
        help   1.2Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    .adjust_abundance_se: no visible binding for global variable ‘x’
    .aggregate_duplicates_se: no visible binding for global variable
      ‘group_name’
    .aggregate_duplicates_se: no visible binding for global variable
      ‘group’
    .deconvolve_cellularity_se: no visible binding for global variable
      ‘X_cibersort’
    .describe_transcript_SE: no visible binding for global variable
      ‘transcript’
    .describe_transcript_SE: no visible binding for global variable
    ...
      predict prop rc read count ref_genome rotated dimensions sample 1
      sample 2 sample a sample b sample_idx samples sdev seurat_clusters
      surv_test temp term test tot tot_filt transcript transcript_upper
      tt_columns update.formula upper value variable vcov web_page where
      with_groups x
    Consider adding
      importFrom("base", "sample")
      importFrom("stats", "AIC", "anova", "coef", "kmeans", "logLik",
                 "predict", "update.formula", "vcov")
    to your NAMESPACE file.
    ```

# tidyseurat

<details>

* Version: 0.8.0
* GitHub: https://github.com/stemangiola/tidyseurat
* Source code: https://github.com/cran/tidyseurat
* Date/Publication: 2024-01-10 04:50:02 UTC
* Number of recursive dependencies: 206

Run `revdepcheck::revdep_details(, "tidyseurat")` for more info

</details>

## In both

*   checking whether package ‘tidyseurat’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ttservice’ was built under R version 4.3.1
      Warning: package ‘SeuratObject’ was built under R version 4.3.1
      Warning: package ‘sp’ was built under R version 4.3.1
      Warning: package ‘dplyr’ was built under R version 4.3.1
      Warning: package ‘tidyr’ was built under R version 4.3.1
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/tidyseurat/new/tidyseurat.Rcheck/00install.out’ for details.
    ```

# tidySingleCellExperiment

<details>

* Version: 1.12.0
* GitHub: https://github.com/stemangiola/tidySingleCellExperiment
* Source code: https://github.com/cran/tidySingleCellExperiment
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 222

Run `revdepcheck::revdep_details(, "tidySingleCellExperiment")` for more info

</details>

## In both

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
       11.           ├─BiocSingular::runPCA(...)
       12.           └─BiocSingular::runPCA(...)
       13.             └─BiocSingular (local) .local(x, ...)
       14.               ├─BiocSingular::runSVD(...)
       15.               └─BiocSingular::runSVD(...)
       16.                 ├─BiocSingular::runSVD(...)
       17.                 └─BiocSingular::runSVD(...)
       18.                   ├─base::do.call(...)
       19.                   └─BiocSingular (local) `<fn>`(...)
       20.                     ├─base::do.call(...)
       21.                     └─irlba (local) `<fn>`(...)

      [ FAIL 1 | WARN 7 | SKIP 0 | PASS 154 ]
      Error: Test failures
      Execution halted
    ```

*   checking whether package ‘tidySingleCellExperiment’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘SingleCellExperiment’ was built under R version 4.3.2
      Warning: package ‘SummarizedExperiment’ was built under R version 4.3.2
      Warning: package ‘MatrixGenerics’ was built under R version 4.3.1
      Warning: package ‘matrixStats’ was built under R version 4.3.1
      Warning: package ‘GenomicRanges’ was built under R version 4.3.1
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
      Warning: package ‘S4Vectors’ was built under R version 4.3.2
      Warning: package ‘IRanges’ was built under R version 4.3.1
      Warning: package ‘GenomeInfoDb’ was built under R version 4.3.2
      Warning: package ‘Biobase’ was built under R version 4.3.1
      Warning: package ‘dplyr’ was built under R version 4.3.1
      Warning: package ‘tidyr’ was built under R version 4.3.1
      Warning: package ‘ttservice’ was built under R version 4.3.1
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/tidySingleCellExperiment/new/tidySingleCellExperiment.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    aggregate_cells,SingleCellExperiment: no visible binding for global
      variable ‘my_id_to_split_by___’
    aggregate_cells,SingleCellExperiment: no visible binding for global
      variable ‘assay_name’
    aggregate_cells,SingleCellExperiment: no visible global function
      definition for ‘join_by’
    aggregate_cells,SingleCellExperiment: no visible binding for global
      variable ‘.feature’
    Undefined global functions or variables:
      .feature assay_name join_by my_id_to_split_by___
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: group_by.Rd:98-100: Dropping empty section \seealso
    ```

# TNBC.CMS

<details>

* Version: 1.18.0
* GitHub: NA
* Source code: https://github.com/cran/TNBC.CMS
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 174

Run `revdepcheck::revdep_details(, "TNBC.CMS")` for more info

</details>

## In both

*   checking whether package ‘TNBC.CMS’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘e1071’ was built under R version 4.3.1
      Warning: package ‘SummarizedExperiment’ was built under R version 4.3.2
      Warning: package ‘MatrixGenerics’ was built under R version 4.3.1
      Warning: package ‘matrixStats’ was built under R version 4.3.1
      Warning: package ‘GenomicRanges’ was built under R version 4.3.1
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
      Warning: package ‘S4Vectors’ was built under R version 4.3.2
      Warning: package ‘IRanges’ was built under R version 4.3.1
      Warning: package ‘GenomeInfoDb’ was built under R version 4.3.2
      Warning: package ‘Biobase’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/TNBC.CMS/new/TNBC.CMS.Rcheck/00install.out’ for details.
    ```

# TOAST

<details>

* Version: 1.16.0
* GitHub: https://github.com/ziyili20/TOAST
* Source code: https://github.com/cran/TOAST
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 118

Run `revdepcheck::revdep_details(, "TOAST")` for more info

</details>

## In both

*   checking whether package ‘TOAST’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘EpiDISH’ was built under R version 4.3.1
      Warning: package ‘limma’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/TOAST/new/TOAST.Rcheck/00install.out’ for details.
    ```

# TOmicsVis

<details>

* Version: 2.0.0
* GitHub: https://github.com/benben-miao/TOmicsVis
* Source code: https://github.com/cran/TOmicsVis
* Date/Publication: 2023-08-28 18:30:02 UTC
* Number of recursive dependencies: 265

Run `revdepcheck::revdep_details(, "TOmicsVis")` for more info

</details>

## In both

*   checking whether package ‘TOmicsVis’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘widgetTools’ was built under R version 4.3.1
      Warning: package ‘DynDoc’ was built under R version 4.3.1
      Warning: package ‘Biobase’ was built under R version 4.3.1
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
      Warning: package ‘e1071’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/TOmicsVis/new/TOmicsVis.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.4Mb
      sub-directories of 1Mb or more:
        data-tables   1.4Mb
        doc           1.9Mb
        help          1.2Mb
    ```

# TVTB

<details>

* Version: 1.28.0
* GitHub: https://github.com/kevinrue/TVTB
* Source code: https://github.com/cran/TVTB
* Date/Publication: 2023-11-02
* Number of recursive dependencies: 192

Run `revdepcheck::revdep_details(, "TVTB")` for more info

</details>

## In both

*   checking Rd files ... WARNING
    ```
    checkRd: (5) VcfBasicRules-class.Rd:133-136: \item in \describe must have non-empty label
    checkRd: (5) VcfBasicRules-class.Rd:137-140: \item in \describe must have non-empty label
    checkRd: (5) VcfBasicRules-class.Rd:141-146: \item in \describe must have non-empty label
    checkRd: (5) VcfBasicRules-class.Rd:147-151: \item in \describe must have non-empty label
    checkRd: (5) VcfBasicRules-class.Rd:161-164: \item in \describe must have non-empty label
    checkRd: (5) VcfBasicRules-class.Rd:165-168: \item in \describe must have non-empty label
    checkRd: (5) VcfBasicRules-class.Rd:179-192: \item in \describe must have non-empty label
    checkRd: (5) VcfBasicRules-class.Rd:193-201: \item in \describe must have non-empty label
    checkRd: (5) VcfFilterRules-class.Rd:85-92: \item in \describe must have non-empty label
    checkRd: (5) VcfFilterRules-class.Rd:93-96: \item in \describe must have non-empty label
    checkRd: (5) VcfFilterRules-class.Rd:97-103: \item in \describe must have non-empty label
    checkRd: (5) VcfFilterRules-class.Rd:104-108: \item in \describe must have non-empty label
    checkRd: (5) VcfFilterRules-class.Rd:120-123: \item in \describe must have non-empty label
    checkRd: (5) VcfFilterRules-class.Rd:124-127: \item in \describe must have non-empty label
    checkRd: (5) VcfFilterRules-class.Rd:134-140: \item in \describe must have non-empty label
    checkRd: (5) VcfFilterRules-class.Rd:141-149: \item in \describe must have non-empty label
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        R     2.1Mb
        doc   2.8Mb
    ```

# ubiquity

<details>

* Version: 2.0.1
* GitHub: https://github.com/john-harrold/ubiquity
* Source code: https://github.com/cran/ubiquity
* Date/Publication: 2023-10-29 20:50:02 UTC
* Number of recursive dependencies: 146

Run `revdepcheck::revdep_details(, "ubiquity")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.3Mb
      sub-directories of 1Mb or more:
        doc     6.6Mb
        ubinc   2.0Mb
    ```

# ushr

<details>

* Version: 0.2.3
* GitHub: https://github.com/SineadMorris/ushr
* Source code: https://github.com/cran/ushr
* Date/Publication: 2020-04-21 18:20:03 UTC
* Number of recursive dependencies: 92

Run `revdepcheck::revdep_details(, "ushr")` for more info

</details>

## In both

*   checking whether package ‘ushr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘dplyr’ was built under R version 4.3.1
      Warning: package ‘tidyr’ was built under R version 4.3.1
      Warning: package ‘ggplot2’ was built under R version 4.3.1
    See ‘/ggally/revdep/checks.noindex/ushr/new/ushr.Rcheck/00install.out’ for details.
    ```

# velociraptor

<details>

* Version: 1.12.0
* GitHub: https://github.com/kevinrue/velociraptor
* Source code: https://github.com/cran/velociraptor
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 237

Run `revdepcheck::revdep_details(, "velociraptor")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘velociraptor-Ex.R’ failed
    The error most likely occurred in:

    > ### Name: embedVelocity
    > ### Title: Project velocities onto an embedding
    > ### Aliases: embedVelocity embedVelocity,ANY-method
    > ###   embedVelocity,SingleCellExperiment-method
    >
    > ### ** Examples
    >
    ...
    To search for alternate channels that may provide the conda package you're
    looking for, navigate to

        https://anaconda.org

    and use the search bar at the top of the page.


    Error: one or more Python packages failed to install [error code 1]
    Execution halted
    ```

*   checking tests ...
    ```
      Running ‘testthat.R’
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      Backtrace:
          ▆
       1. ├─velociraptor::scvelo(...) at test-scvelo.R:70:5
       2. └─velociraptor::scvelo(...)
       3.   └─velociraptor (local) .local(x, ...)
       4.     └─basilisk::basiliskRun(...)
       5.       └─basilisk::basiliskStart(...)
       6.         └─basilisk::obtainEnvironmentPath(env)
       7.           └─basilisk::setupBasiliskEnv(...)
       8.             └─reticulate::conda_install(...)
       9.               └─reticulate:::stopf(fmt, result)

      [ FAIL 6 | WARN 14 | SKIP 0 | PASS 0 ]
      Error: Test failures
      Execution halted
    ```

*   checking whether package ‘velociraptor’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘SummarizedExperiment’ was built under R version 4.3.2
      Warning: package ‘MatrixGenerics’ was built under R version 4.3.1
      Warning: package ‘matrixStats’ was built under R version 4.3.1
      Warning: package ‘GenomicRanges’ was built under R version 4.3.1
      Warning: package ‘BiocGenerics’ was built under R version 4.3.1
      Warning: package ‘S4Vectors’ was built under R version 4.3.2
      Warning: package ‘IRanges’ was built under R version 4.3.1
      Warning: package ‘GenomeInfoDb’ was built under R version 4.3.2
      Warning: package ‘Biobase’ was built under R version 4.3.1
    Found the following additional notes/warnings:
      Non-staged installation was used
    See ‘/ggally/revdep/checks.noindex/velociraptor/new/velociraptor.Rcheck/00install.out’ for details.
    ```

# vidger

<details>

* Version: 1.22.0
* GitHub: https://github.com/btmonier/vidger
* Source code: https://github.com/cran/vidger
* Date/Publication: 2023-10-24
* Number of recursive dependencies: 128

Run `revdepcheck::revdep_details(, "vidger")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.5Mb
      sub-directories of 1Mb or more:
        data   3.0Mb
        doc    6.2Mb
    ```

# WOTPLY

<details>

* Version: 0.1.0
* GitHub: NA
* Source code: https://github.com/cran/WOTPLY
* Date/Publication: 2022-09-12 07:43:01 UTC
* Number of recursive dependencies: 83

Run `revdepcheck::revdep_details(, "WOTPLY")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘sna’
      All declared Imports should be used.
    ```
